### ISSUE TITLE
Java 11 project gives warning: SneakyThrowsRemover caused an Exception

### ISSUE THREAD

**Issue by dnno** [mplushnikov/lombok-intellij-plugin#520]
### Short description
I've created a simple Java 11 based project using Lombok, when I build it using Intellij I get a warning:

```
Warning:java: Error during the transformation of 'Main$Test'; post-compiler 'lombok.bytecode.SneakyThrowsRemover' caused an exception: java.lang.UnsupportedOperationException
```

### Expected behavior
No warning should be issued

### Version information

 * **IDEA Version:** 2018.2
 * **JDK Version:** JDK 11
openjdk 11-ea 2018-09-25
OpenJDK Runtime Environment 18.9 (build 11-ea+25)
OpenJDK 64-Bit Server VM 18.9 (build 11-ea+25, mixed mode)
 * **OS Type & Version:** Mac OS X, 10.13.6
 * **Lombok Plugin Version:** 0.19-LATEST-EAP-SNAPSHOT
 * **Lombok Dependency Version:** 1.18.2

### Steps to reproduce

1. Set up a Java 11 environment
2. Clone this repository: https://github.com/dnno/java11-lombok-plugin
3. When you build the project (or run a "Rebuild") you should see the warning

### Additional information
A Maven build does not produce this warning.

**Comment by dnno** [comment]
Has anyone had the chance to look at this?

**Comment by VagrantBlackbird** [comment]
Hey, we looked at this with a number of people here, including some of the Lombok authors. 

First of all, the warning does not appear for you because you have maven warnings disabled in your repository. Adding:
```
			<configuration>
					<showWarnings>true</showWarnings>
				</configuration>
```
should solve that problem.

Secondly, we found out that this is actually a lombok bug related to the concept of Nest-based access control. We're going to file a bug at Lombok and we have an idea for a fix as well.
